_inc/client/plans/single-product-backup.scss

```diff
@@ -1,3 +1,21 @@
+@import '../scss/variables/_colors.scss';
+
+.plans-section__header {
+	color: #23282d;
+	text-align: center;
+	font-family: Roboto, sans-serif;
+	font-weight: normal;
+	font-size: 22pt;
+}
+
+.plans-section__subheader {
+	color: #747474;
```

Comment:
Let's make use of the color definitions in `../scss/variables/_colors.scss` and replace it with:

```scss
color: $gray-text-min;
```